.github/workflows/smoke.yml

```diff
@@ -16,6 +16,8 @@ env:
   CARGO_TERM_COLOR: always
   IMAGE: wordpress
   TAG: 6.1.1
+  NYDUS_STABLE_VERSION: v2.2.3
```

Comment:
Let's fix https://github.com/dragonflyoss/nydus/actions/runs/6715346171/job/18249975611?pr=1461 first, maybe the better way is to get the latest stable version name using GitHub API.

Reply:
Maybe see https://api.github.com/repos/Dragonflyoss/nydus/releases/latest